cmd/loop/liquidity.go

```diff
@@ -173,6 +174,149 @@ func setRule(ctx *cli.Context) error {
 	return err
 }
 
+var setParamsCommand = cli.Command{
+	Name:        "setparams",
+	Usage:       "update the parameters set for the liquidity manager",
+	Description: "Updates the parameters set for the liquidity manager.",
+	Flags: []cli.Flag{
+		cli.Float64Flag{
+			Name: "maxswapfee",
+			Usage: "the maximum percentage of swap volume we are " +
+				"willing to pay in server fees, expressed as " +
+				"a decimal (eg, 0.01 = 1%)",
+		},
+		cli.Float64Flag{
+			Name: "maxroutingfee",
+			Usage: "the maximum percentage of off-chain payment " +
+				"volume that are are willing to pay in " +
+				"routing fees, expressed as a decimal (eg, " +
+				"0.01 = 1%).",
+		},
+		cli.Float64Flag{
+			Name: "maxprepayfee",
+			Usage: "the maximum percentage of off-chain prepay " +
+				"volume that are are willing to pay in " +
+				"routing fees, expressed as a decimal (eg, " +
+				"0.01 = 1%).",
```

Comment:
What do you think about my outstanding comment that a float may easily lead to the wrong number of zeroes? Isn't it better ux to accept a (float) percentage?

Reply:
Ah yes definitely, I misunderstood the original comment. Updated to float percentage, much nicer!